UnityProject/Assets/Scripts/Systems/Faith/FaithData.cs

```diff
@@ -38,6 +39,7 @@ public void RemoveAllMembers()
 
 		public void SetupFaith()
 		{
+			Loggy.Log("[FaithData/SetupFaith] Setting up faith data for " + Faith.FaithName);
```

Comment:
log

Reply:
It's left there on purpose to keep track of what the faith system is doing